stripper/ze_touhou_gensokyo_o4.cfg

```diff
@@ -1,3 +1,31 @@
+;Prevent a strafe jump around an invis wall in boss arena
+add:
+{
+	"origin" "-11891 -5881.5 4622"
+	"angles" "0 0 0"
+	"model" "*4"
+	"UseLandmarkAngles" "1"
+	"target" "bosstar"
+	"StartDisabled" "0"
+	"spawnflags" "4097"
+	"CheckDestIfClearForPlayer" "0"
+	"classname" "trigger_teleport"
+}
+
+;Make ZMs in Boss arena at end get TPed with other ZMs rather than avoiding all TPs
+entity
```

Comment:
entity?

Reply:
works with entity (cause I had tested this before hand), changed for consistancy with other strippers. Also added filter to zombie team, though I think it should have worked fine without it (just in case).